rita_client/src/heartbeat/mod.rs

```diff
@@ -114,7 +112,6 @@ pub fn send_heartbeat_loop() {
             error!("Heartbeat loop thread panicked! Respawning {:?}", e);
             if Instant::now() - last_restart < Duration::from_secs(60) {
                 error!("Restarting too quickly, leaving it to auto rescue!");
-                system.stop_with_code(121)
```

Comment:
Without this we won't stop the whole application at this point. The new solution will be somewhat complicated since I think we'll need thread handles for the other loops. Just modify the messages here to indicate that we simply log this error. After all I can't find any instances of this log in graylog so it's hard to even know if this is ever being called anyways.